C/plugins/storage/sqlite/common/readings.cpp

```diff
@@ -741,36 +761,39 @@ int localNReadingsTotal;
 			{
 				readingsId = readCatalogue->getReadingReference(this, asset_code);
 
-				if (readingsId >= localNReadingsTotal)
+				if (readingsId == -1)
 				{
-					//# FIXME_I:
-					localNReadingsTotal = readingsId + 1;
-					readingsStmt.resize(localNReadingsTotal, nullptr);
-
-					//# FIXME_I
-					Logger::getLogger()->setMinLevel("debug");
-					Logger::getLogger()->debug("xxx4 readingsStmt resize size :%d: idx :%d: ", localNReadingsTotal, readingsId);
-					Logger::getLogger()->setMinLevel("warning");
+					Logger::getLogger()->warn("Impossible to insert the row for the asset_code :%s: into the readings, row ignored.", asset_code);
```

Comment:
Rather than say "Impossible to..." can we make it say "It was not possible to ...". Also if we could say why it was not possible that would be useful.

Reply:
Text adapted.
that piece of code state that was not possible to insert the row (as a WARNING) the function called immediately before will log the ERROR for the specific issue, log sample: 

``` 
ERROR: attachNewDb - It was not possible to attach the db :/home/foglamp/Development/fledge/data/readings_10.db: to an idle connection, error :too many attached databases - max 10:
WARNING: appendReadings - It was not possible to insert the row for the asset_code :msec_011_OK: into the readings, row ignored.
```